neqo-http3/src/stream_type_reader.rs

```diff
@@ -108,9 +110,11 @@ impl NewStreamHeadReader {
             reader, stream_id, ..
         } = self
         {
+            // This type only exists to read at most two varints (= 16 bytes) from the stream.
+            let mut buf = [0; 2 * size_of::<u64>()];
```

Comment:
```suggestion
            let mut buf = [0; 16];
```

varints aren't really `u64`, so why not just write 16 and be done.